process/working-group-lifecycle.md

```diff
@@ -8,14 +8,29 @@ However, if no existing effort covers the topic feel free to make use of the exi
 
 Please note that running a WG successfully requires an extended effort. Be prepared to commit at least several hours per week for a year or more.
 
-The WG Life Cycle begins with establishing commitments from other interested contributors then making a proposal to the TAC. If accepted, the WG will begin as `Incubating`. This is a boot strapping phase for the group to adopt OpenSSF community policies and norms. Once those are met the group becomes `Active` and will be fully recognized in OpenSSF. Either upon completing its objectives or no longer making forward progress a project will move to `End-of-Life`. The details for each phase follow.
+The WG Life Cycle begins with establishing commitments from other interested contributors then making a proposal to the TAC. If accepted, the WG will begin as `Samdbox`. This is a boot strapping phase for the group to define and document what their goals are. 
+adopt OpenSSF community policies and norms. Once those are met the group becomes `Active` and will be fully recognized in OpenSSF. Either upon completing its objectives or no longer making forward progress a project will move to `End-of-Life`. The details for each phase follow.
```

Comment:
It looks like there's a dangling sentence fragment here "adopt OpenSSF community policies and norms."

Reply:
Also let's swap `Active` for `Graduated` and `End-of-Life` for `Archived`?

Reply:
Done.